src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,62 @@
-export const Tabs = () => {};
+import React from 'react';
+
+interface Tab {
+  id: string,
+  title: string,
+  content: string,
```

Comment:
Better create a separate file for all types and import them in order not to duplicate them